rust/src/dataset.rs

```diff
@@ -337,6 +338,31 @@ impl Dataset {
         })
     }
 
+    /// Create a new version of [`Dataset`] from a collection of fragments.
+    pub async fn create_version_from_fragments(
```

Comment:
could we use this to manually create a version after distributed writes?

Reply:
Yes, this is exactly the purpose of exposing this function as pub.